interface/main/tabs/main.php

```diff
@@ -1,5 +1,6 @@
 <?php
-Use Esign\Api;
+use Esign\Api;
+use OpenEMR\Core\Header;
```

Comment:
Would it be bad if we kept the use statement always above the Header call since this is a 1 time thing and that developers will mimick this code (and will forget the use if it isn't above it)?